app/services/solidus_pay_tomorrow/client/create_order_service.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module SolidusPayTomorrow
+  module Client
+    class CreateOrderService < SolidusPayTomorrow::Client::BaseService
+      attr_reader :order
+
+      CREATE_ENDPOINT = 'api/application/ecommerce/orders'
+      # TODO: Update URI once implemented
+      WEBHOOK_URL = 'https://domain.com/webhooks/:type'
```

Comment:
Considering the URL will depend on the domain, probably it shouldn't be a constant, but a method.

Reply:
**Confirmation:** Will this be fine?

```ruby
def webhook_url(type)
  "https://domain.com/webhooks/#{type}"
end
```

Reply:
Fixed in https://github.com/nebulab/solidus_pay_tomorrow/pull/7/commits/75dc5280c554494a2fd5784ac6c45656b06e9fdc